### PR TITLE
Label input elements

### LINKED ISSUE #2047: a11y: add labels to input elements
```
$ grep -r 'require-input-label' .lint-todo/ | egrep -o '"filePath":"[^"]+"' | sort -u
"filePath":"addon/components/boolean-check.hbs"
"filePath":"addon/components/course-header.hbs"
"filePath":"addon/components/course-materials.hbs"
"filePath":"addon/components/course-overview.hbs"
"filePath":"addon/components/course-rollover.hbs"
"filePath":"addon/components/course-sessions.hbs"
"filePath":"addon/components/course-visualize-instructors.hbs"
"filePath":"addon/components/dashboard-calendar.hbs"
"filePath":"addon/components/leadership-search.hbs"
"filePath":"addon/components/learningmaterial-manager.hbs"
"filePath":"addon/components/my-materials.hbs"
"filePath":"addon/components/new-learningmaterial.hbs"
"filePath":"addon/components/new-session.hbs"
"filePath":"addon/components/offering-form.hbs"
"filePath":"addon/components/search-box.hbs"
"filePath":"addon/components/session-copy.hbs"
"filePath":"addon/components/session-overview.hbs"
"filePath":"addon/components/session/postrequisite-editor.hbs"
"filePath":"addon/components/sessions-grid-offering.hbs"
"filePath":"addon/components/taxonomy-manager.hbs"
"filePath":"addon/components/time-picker.hbs"
"filePath":"tests/dummy/app/templates/login.hbs"
```

### PR DESCRIPTION
**Pull request by stopfstedt**
refs #2047

### CODE REVIEW

**Comment by stopfstedt** [comment]
i ran out of time today, but this is good to go.

File: addon/components/course-rollover.hbs [hunk 1]
```diff
@@ -3,136 +3,144 @@
...
-        <Course::RolloverDatePicker
-          @course={{@course}}
-          @value={{this.startDate}}
-          @change={{set this.startDate}}
-        />
+        <input
+          id="title-{{templateId}}"
+          type="text"
+          value={{this.title}}
+          {{on "input" (pipe (pick "target.value" (set this.title)) (fn this.addErrorDisplayFor "title"))}}
```
Comment (jrjohnson) [inline addon/components/course-rollover.hbs:27]: `pipe` is fine, but It think it's unnecessary here. Adding two `{{on "input"` handlers should work as well and is a bit easier to read.

File: addon/components/print-course.hbs [hunk 2]
```diff
@@ -345,117 +194,289 @@
...
+          <div class="inline-label-data-block">
+            <label for="supplemental-curriculum-{{templateId}}">
+              {{t "general.supplementalCurriculum"}}:
+            </label>
+            <div>
+              <input
+                id="supplemental-curriculum-{{templateId}}"
+                type="checkbox"
+                checked={{session.supplemental}}
+                disabled="disabled"
```
Comment (jrjohnson) [inline addon/components/print-course.hbs:241]: Things I learned `disabled="disabled"` is the most semantically valid way to specify this 🤔

**Review by jrjohnson — APPROVED** [review]

File: addon/components/course-rollover.hbs (hunk 1 above)
Comment (stopfstedt) [reply addon/components/course-rollover.hbs]: :heavy_check_mark:

File: addon/components/print-course.hbs (hunk 2 above)
Comment (stopfstedt) [reply addon/components/print-course.hbs:241]: :+1:

**Pull request merged**